tests/ui/parser/attribute/invalid-delimeter-ice-146808.rs

```diff
@@ -0,0 +1,13 @@
+// regression test for #146808
+//@ proc-macro: all_spans_same.rs
+//@ run-rustfix
+// the fixed program is still broken, but rustfix didn't crash!
+// that's what we want to test here.
+//@ rustfix-dont-test-fixed
+
+extern crate all_spans_same;
+
+#[all_spans_same::all_spans_same]
+//~^ ERROR wrong meta list delimiters
+#[allow{}]
```

Comment:
crashes, because we suggest changing {} to () but the suggestion consists of two parts `(` and `)` with both the same span